README.md

```diff
@@ -4,22 +4,48 @@ In this repository, we collect example scripts that can be useful for the team a
 
 ## Prerequisites
 
-TBD
+To run python scripts inside `examples` directory you should first run the installer script:
+
+```bash
+$ bin/install
+```
+
+This will handle installing the required packages into virtualenv.
+
+### Authorization
+
+In order to authorize the requests, please add your credentials in a `JSON` format into the `config.json` file created by install script.
+
+Note: You need to authorize yourself through OAuth to get `access_token` and `refresh_token`. if token gets expired, the script will handle fetching a new one. here is how to use [OAuth](https://community.canvaslms.com/t5/The-Product-Blog/Connecting-Studio-OAuth-via-Postman/ba-p/259739)
```

Comment:
I would rather say this:

You can generate the necessary credentials as described [here](https://community.canvaslms.com/t5/The-Product-Blog/Connecting-Studio-OAuth-via-Postman/ba-p/259739).

Once you have them in config.json, you won't have to deal with them anymore: the example scripts automatically handle renewing expired tokens.

Reply:
done